content/en/docs/concepts/security/hardening-guide/scheduler.md

```diff
@@ -0,0 +1,90 @@
+| `authentication-tolerate-lookup-failure` | Set to `false` to make sure invalid authentication configurations do not lead to requests passing off as anonymous                  |
+| `authentication-skip-lookup`             | This should be set to `false` to make sure all missing authentication configuration falls back to the authentication kubeconfig.    |
+| `authorization-always-allow-paths`       | These paths should respond with data that is appropriate for anonymous authorization. Defaults to `/healthz,/readyz,/livez`.        |
+{{</table>}}
+
+### Address configuration options
+{{<table caption="Address Configurations">}}
+| Configuration            | Security hardening advice                                                                                                                                                          |
+| ------------------------ | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `bind-address`           | In most cases, the kube-scheduler does not need to be externally accessible. Setting the bind address to `localhost` is a secure practice.                                         |
+| `permit-address-sharing` | Set this configuration to `false` to  disable connection sharing through `SO_REUSEADDR`. `SO_REUSEADDR` can lead to reuse of terminated connections that are in `TIME_WAIT` state. |
+| `permit-port-sharing`    | Default `false`. Use the default unless you are confident you understand the security implications.                                                                                |
+{{</table>}}
+
+### TLS configuration options
+{{<table caption="Address Configurations">}}
+| Configuration                  | Security hardening advice                                                                                                                                                                                                          |
+| ------------------------------ | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `requestheader-client-ca-file` | Generally do not depend on authorization being already done for incoming requests. Always provide the root certificate bundle. This allows authorization to happen on each incoming request through `requestheader-allowed-names`. |
+| `tls-cipher-suites`            | Always provide a list of preferred cipher suites. This ensures encryption never happens with insecure cipher suites.                                                                                                               |
+{{</table>}}
+
+## Scheduling configurations for custom schedulers
+This section covers security hardening when using custom schedulers.
```

Comment:
```suggestion
This section covers security hardening when using custom schedulers based on the Kubernetes
scheduling code.
```

Do we have any advice for securing a custom scheduler written in something else (eg: shell, Python)?